packages/ui/components/form/color-picker/colorpicker.tsx

```diff
@@ -27,8 +29,8 @@ const ColorPicker = (props: ColorPickerProps) => {
             />
           </Popover.Trigger>
         </div>
-        <Popover.Portal>
-          <Popover.Content align="center" sideOffset={10}>
+        <Popover.Portal container={props.container}>
```

Comment:
Container on portal allows Popover to be descendant of the container. 
We need to make popover descendant of the Dialog Content so that it can be focussed.
Here is a related issue https://github.com/radix-ui/primitives/issues/922